cppapi/server/dserverclass.cpp

```diff
@@ -1845,12 +1845,15 @@ void DServerClass::device_factory(const Tango::DevVarStringArray *devlist_ptr)
 //
 // Create device and add it into the device list
 //
+        auto dserver = new DServer(this,
```

Comment:
We should not use auto if we want to be able to compile on old windows compilers.